### ISSUE TITLE
Station logs overflowing storage and not being properly rotated

### ISSUE THREAD

**Issue by aripisprecer** [AzuraCast/AzuraCast#662]
**Installation method**
"Traditional" installation method (LXC container)
Ubuntu 18.04

**Describe the bug**
I am running AzuraCast on a LXC container with 5GB storage and i don't use liquidsoap and autodj, I use olny shoutcast and icecast as streaming servers.

After about one week, the storage is getting full and this causes other problems: nginx, php, redis, mysql are going down and i cannot acces the azuracast interface.

I've checked the disk space, and the most occupied space is on /var/azuracast/stations/.....station... log files , than on the /tmp/ there are a lot of log files from shoutcast and supervisor.

I've also checked the crontab fort the azuracast user, and there seems to be no reorg for the logfiles.

**Comment by SlvrEagle23** [comment]
@aripisprecer I looked into it, and while Shoutcast does appear to have proper log rotation for log files, Icecast does not, and the previous default setting for logging was "Info", which indeed may be too verbose for your average production station and may be overflowing the log folders. I've switched it to be "Warnings" for production instances and "Info" for development only.

Implementing log rotation for traditional installs will definitely be a little trickier of an item and will take more time. Hopefully this helps keep the Icecast logs from piling up on you at least.
